pkgs/development/libraries/libbson/default.nix

```diff
@@ -0,0 +1,27 @@
+{ autoconf, automake, fetchgit, libtool, perl, stdenv, which }:
+
+let
+  version = "1.1.10";
+in
+
+stdenv.mkDerivation (rec {
+  name = "libbson-${version}";
+
+  src = fetchgit {
+    url = "git://github.com/mongodb/libbson";
+    rev = "refs/tags/${version}";
+    sha256 = "9ad2df5238cdb52397daac119f3647e25fdca9165333adb30282af984c15f03a";
+  };
+
+  buildInputs = [ autoconf automake libtool perl which ];
+
+  configureScript = "./autogen.sh";
+
+  meta = with stdenv.lib; {
+    description = "Provides useful routines related to building, parsing, and iterating BSON documents. It is a useful base for those wanting to write high-performance C extensions to higher level languages such as python, ruby, or perl.";
```

Comment:
Please change "description" according to http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes. Use "longDescription" (in addition to "description") if you want more text.